plugins/module_utils/turbo/lookup_server.py

```diff
@@ -0,0 +1,142 @@
+import argparse
```

Comment:
Do we need to create another server? I was under the impression it was possible to reuse the same for both module and lookup plugins.

Reply:
merged into a single server